assignment/index.html

```diff
@@ -63,18 +63,62 @@
       ===================== */
 
       var jsonToCsv = function(json) { console.log(json); };
-
-
-      var addMarkers = function(map) {};
+      // Preloaded healthcenter JSON
+      var healtharray = [["LNG","LAT","OBJECTID","NAME","ORGANIZATION","ZIP","PHONE","WEBSITE","DENTAL_PHONE","FULL_ADDRESS"]];
+
+      for (var i=0;i<healthCenters.length; i++){
+           var healthInfo =[healthCenters[i].LNG,
+                            healthCenters[i].LAT,
+                            healthCenters[i].OBJECTID,
+                            healthCenters[i].NAME,
+                            healthCenters[i].ORGANIZATION,
+                            healthCenters[i].ZIP,
+                            healthCenters[i].PHONE,
+                            healthCenters[i].WEBSITE,
+                            healthCenters[i].DENTAL_PHONE,
+                            healthCenters[i].FULL_ADDRESS];
+                            healtharray.push(healthInfo)
+                          };
+      console.log(healtharray);
+
+      var Icon1 = L.icon({
+    iconUrl: '/Users/wangzixuan/Desktop/MUSA611-JS/cpln692-week2/assignment/images/marker-icon.png',
```

Comment:
Absolute path URLs are best avoided because they make it difficult to use this code on anyone elses computer.

If you turn it relative, and do this:

iconUrl: 'images/marker-icon.png'

it works anywhere.